internal/hcs/process.go

```diff
@@ -113,9 +113,9 @@ func (process *Process) processSignalResult(ctx context.Context, err error) (boo
 
 // Signal signals the process with `options`.
 //
-// For LCOW `guestrequest.SignalProcessOptionsLCOW`.
+// For LCOW `protocol.SignalProcessOptionsLCOW`.
```

Comment:
guestresource.SignalProcessOptionsLCOW and guestresource.SignalProcessOptionsWCOW?